### PR TITLE
build(js): don't ship built javascript assets

### PR DESCRIPTION
**Pull request by mejo-**
Not needed since it's not a shipped app.

### CODE REVIEW

File: .github/workflows/node.yml
Comment (nickvergessen) [inline .github/workflows/node.yml:1]: This should still be kept. Ensures that compiling still works and highlights errors on CI otherwise.

**Review by nickvergessen — CHANGES_REQUESTED** [review]

File: .github/workflows/node.yml
Comment (mejo-) [reply .github/workflows/node.yml:1]: Don't you think that running the Cypress test is enough to ensure this?

Comment (nickvergessen) [reply .github/workflows/node.yml:1]: Not a fan of implicit testing :P

Comment (mejo-) [reply .github/workflows/node.yml:1]: Ok ok, added back `.github/workflows/node.yml` :wink:

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**